### PR TITLE
14-separate-sites

### LINKED ISSUE #14: Separate spaces for official and community docs
The story:

As a user I want a clear distinction between documentation provided and supported by ACCESS-NRI, and documentation that is provided by the community.

As a member of the ACCESS-NRI I want it to be clear what documentation is supported by the NRI, so the community know where they can go if they have problems with NRI provided documentation.

A possible solution:

The [PR preview feature](https://github.com/ACCESS-NRI-Community/ACCESSHub/issues/11) shows that it is possible for two separate mkdocs sites to exist in their own subdirectory. This may be a solution to this issue: build and deploy the `main` branch to https://access-nri-community.github.io/ACCESSHub/Community and have another branch, `access-nri`, that deploys to https://access-nri-community.github.io/ACCESSHub/ACCESS-NRI.

Add some protection rules to the `access-nri` branch and that is a relatively simple solution. The `access-nri` branch can have different theme to differentiate it, and modified CI so that it deploys to the correct location.

### PR DESCRIPTION
**Pull request by ccarouge**
Closes #14

This is to create the 2 sites for ACCESS-Hub. I leave the PR #37 up so we can use that for trial and error if needed.

### CODE REVIEW

**Review by aidanheerdegen — APPROVED** [review]

File: mkdocs-community.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    - Ocean Sea-ice: evaluation_and_analysis/ocean_seaice.md
+    - Atmosphere: evaluation_and_analysis/atmosphere.md
+    - Machine Learning: evaluation_and_analysis/machine_learning.md
+  - Training resources:
+    - General technical: training/general_technical.md
+    - Modelling: training/modelling.md
+    - Analysis: training/analysis.md
+  - How to contribute: 
+    - How to contribute: help/how_to_contribute.md
+    - Setup: help/contribution_setup.md
```
[the remark this answers is not in the document]
Comment (ccarouge) [reply mkdocs-community.yml:73]: It works when building locally. I'll see after pushing for the review if the preview is still broken.

File: access-nri/mkdocs-nri.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      emoji_index: !!python/name:materialx.emoji.twemoji
+      emoji_generator: !!python/name:materialx.emoji.to_svg
+  - md_in_html
+  - pymdownx.details
+  - pymdownx.superfences
+
+  
+# Navigation
+nav:
+  - index.md
```
[the remark this answers is not in the document]
Comment (ccarouge) [reply access-nri/mkdocs-nri.yml:46]: Added to mkdocs_community.yml

File: access-nri/mkdocs-nri.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    - Land: model_components/land.md
+    - Ocean: model_components/ocean.md
+    - Sea Ice: model_components/sea-ice.md
+    - Atmosphere: model_components/atmosphere.md
+    - Coupler: model_components/coupler.md
+  - Documentation and Training:
+    - Technical: docs_and_training/technical.md
+    - Analysis: docs_and_training/analysis.md
+    - Data: docs_and_training/data.md
+    - Policies : docs_and_training/policies.md
```
[the remark this answers is not in the document]
Comment (ccarouge) [reply access-nri/mkdocs-nri.yml:62]: Moved

File: access-nri/mkdocs-nri.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    - Tools: model_evaluation/tools.md
+    - Tutorials: model_evaluation/tutorials.md
+    - Background information: model_evaluation/background.md
+  - Working Groups:
+    - Ocean-Sea Ice (COSIMA): working_groups/ocean_seaice.md
+    - Atmosphere: working_groups/atmosphere.md
+    - Coupled Models: working_groups/coupled.md
+    - Land: working_groups/land.md
+    - Technical: working_groups/technical.md
+    - Next Generation Methods: working_groups/ngms.md
```
[the remark this answers is not in the document]
Comment (ccarouge) [reply access-nri/mkdocs-nri.yml:73]: moved.

File: .github/workflows/preview.yml [hunk 5]
```diff
@@ -43,7 +43,8 @@ jobs:
       - name: Build
         # builds to "site" directory by default
         run: |
-          mkdocs build 
+          mkdocs build -f mkdocs-community.yml -d site
+          (cd access-nri && mkdocs build -f mkdocs-nri.yml -d ../site/ACCESS-NRI)
```
[the remark this answers is not in the document]
Comment (ccarouge) [reply .github/workflows/preview.yml:47]: Added to publish.yml

File: mkdocs-community.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ccarouge) [reply mkdocs-community.yml:81]: It seems the problem is still here. Looking at the gh-pages branch, it seems mkdocs doesn't handle the change of case well. There is still a "Help" folder when it should be "help". I'll rewind to the uppercase one for now then.

**Comment by aidanheerdegen** [comment]
It should be possible to remove selected stuff from the `gh-pages` branch. You can add `clean` to the [deploy action](https://github.com/marketplace/actions/deploy-to-github-pages) but that will delete the PR reviews. So could add it and then remove it again once the branch is cleaned, or delete stuff by hand by directly pushing to `gh-pages`

**Comment by ccarouge** [comment]
I found the issue. The Mac OS is case-insensitive! Doing it on Gadi worked as expected...

**Pull request merged**